bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/ghost/GhostReadoutTime.java

```diff
@@ -32,18 +27,10 @@ private static double calcReadOut(int binx, int biny, int readRate) {
      */
 
     public static double getReadoutOverhead(Config config) {
-        final GhostType.ReadMode readMode                = (GhostType.ReadMode) config.getItemValue(GhostType.ReadMode.KEY);
-        final GhostType.DetectorManufacturer detMan      = (GhostType.DetectorManufacturer) config.getItemValue(GhostType.DetectorManufacturer.KEY);
-        final GhostType.Binning              xBin        = (GhostType.Binning) config.getItemValue(InstGhost.X_BIN_KEY);
-        final GhostType.Binning              yBin        = (GhostType.Binning) config.getItemValue(InstGhost.Y_BIN_KEY);
-        return calcReadOut(xBin.getValue(),yBin.getValue(), readMode.getReadRate())/1000000.0;
-        /*System.out.println("readmode displayValue" + readMode.displayValue());
-        System.out.println("detMan value" + detMan.displayValue() + "  " + detMan.getManufacter());
-        System.out.println("bining, x: " + xBin.getValue() + " y: " + yBin.getValue());
-        final GhostReadoutKey key = new GhostReadoutKey(readMode, yBin.getValue(), xBin.getValue(), detMan);
-        final Double d = map.get(key);
-         */
-
+        final GhostReadNoiseGain readMode                = (GhostReadNoiseGain) config.getItemValue(null);
+        // I understand RED is the limiting factor
+        final GhostBinning              bin        = (GhostBinning) config.getItemValue(Ghost.RED_BINNING_KEY());
+        return calcReadOut(bin.getSpectralBinning(), bin.getSpectralBinning(), readMode.getReadRate())/1000000.0;
```

Comment:
Will / can this be switched to use `GhostCameras.readout` ?

Reply:
This is not currently used, I have it as reference but should be unified